content/attacks/posts/2020-09-25-KuCoin.md

```diff
@@ -0,0 +1,56 @@
+---
+date: 2020-09-26
+custodians: KuCoin
+categories: Custodian
+title: KuCoin Suffers $281 Million Hack
+---
+
+## Summary
+
+On September 26, 2020, KuCoin, a Singapore-based cryptocurrency exchange, [experienced a significant security breach](https://www.kucoin.com/ru/news/en-kucoin-security-incident-update), resulting in the theft of approximately $281 million worth of cryptocurrencies. [The attackers obtained the private keys to the exchange’s hot wallets](https://www.coindesk.com/markets/2020/09/26/over-280m-drained-in-kucoin-crypto-exchange-hack/). As a [result](https://hacken.io/insights/kucoin-september-2020-hack-hacken-research/), 1,008 BTC was stolen, along with 14,713 BSV, 26,733 LTC, 9,588,383 XLM, Omni, and EOS-based tether (USDT) worth $14 million, $153 million in ether and ERC20s, and over 18 million XRP. The hackers [sold the stolen cryptocurrency](https://hacken.io/insights/kucoin-september-2020-hack-hacken-research/) from their addresses on decentralized exchanges and anonymized the stolen cryptocurrencies through the mixing services. [The incident caused a temporary drop in the price of KuCoin's exchange token KCS by 14%, to $0.86](https://www.coindesk.com/markets/2020/09/26/over-280m-drained-in-kucoin-crypto-exchange-hack/). However, by November 11, 2020, [KuCoin was able to recover all the stolen assets](https://cointelegraph.com/news/kucoin-ceo-says-insurance-covered-16-of-losses-from-285m-hack-in-2020), so users were unaffected by the hack: [84% were regained by on-chain tracking, contract upgrade and judicial recovery](https://cointelegraph.com/news/kucoin-recovered-84-of-stolen-crypto-after-280m-hack-says-co-founder); [the remaining 16% were covered by the KuCoin insurance funds](https://www.kucoin.com/ru/news/en-a-letter-from-kucoin-ceo-2020-2021-and-beyond).
```

Comment:
> However, by November 11, 2020, [KuCoin was able to recover all the stolen assets](https://cointelegraph.com/news/kucoin-ceo-says-insurance-covered-16-of-losses-from-285m-hack-in-2020), so users were unaffected by the hack: [84% were regained by on-chain tracking, contract upgrade and judicial recovery](https://cointelegraph.com/news/kucoin-recovered-84-of-stolen-crypto-after-280m-hack-says-co-founder); [the remaining 16% were covered by the KuCoin insurance funds](https://www.kucoin.com/ru/news/en-a-letter-from-kucoin-ceo-2020-2021-and-beyond).

Don't overuse sources for not as useful information, let's leave first source only